src/lib/components/Section.svelte

```diff
@@ -26,9 +29,17 @@
     justify-content: space-between;
 
     margin-bottom: var(--padding);
+
+    &.noDescription {
```

Comment:
You can spare one assignment here I think

```
&:not(.noDescription) {
  margin-bottom: var(--padding);
}
```

Reply:
thanks!